docs/upgrade.html

```diff
@@ -19,7 +19,7 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h4><a id="upgrade_2_0_0" href="#upgrade_2_0_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, or 1.2.x to 2.0.0</a></h4>
+<h4><a id="upgrade_2_0_0" href="#upgrade_2_0_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x or 1.1.x to 2.0.0</a></h4>
```

Comment:
There's a whole holy war about the oxford comma, so it's very much a matter of opinion, but I personally think that it makes lists clearer, so I'd recommend keeping it. (I'm talking about the comma before "or", which is grammatically not necessary, but sometimes enhances readability)

Reply:
I love the Oxford comma!!!